src/utilities/formatToProposal.ts

```diff
@@ -87,6 +114,16 @@ const formatToProposal = ({
     startTxHash: startTxHash ?? undefined,
     totalVotesWei: abstainedVotesWei.plus(againstVotesWei).plus(forVotesWei),
   };
+
+  if (actions) {
```

Comment:
Actions are required to submit a proposal so there should always be at least 1

Reply:
They are yes, but the `/voters/history/` endpoint does not return any `actions` field which is why this check is here

Reply:
Interesting, that seems like a bug on the endpoint